tests/h/search/index_test.py

```diff
@@ -241,6 +241,23 @@ def test_you_can_filter_annotations_by_thread_ids(self, es_client, factories, in
 
         assert SearchResponseWithIDs([annotation2.id]) == response1
 
+    @pytest.mark.parametrize("quote,query", [
+        ("It is a truth universally acknowledged", "truth"),
+        ("यह एक सत्य सार्वभौमिक रूप से स्वीकार किया जाता है", "सत्य"),
+        ("quick brown fox", "QUICK"),
+    ])
+    def test_match_annotation_by_quote(self, factories, index, search, quote, query):
+        """
+        Verify that the "TextQuoteSelector" selector is indexed as the "quote" field.
+        """
```

Comment:
It doesn't seem to be triggering `make lint` but this'll fail pylint / pep257: Putting a single-line docstring over multiple lines (the quotes need to go on the same line)

Reply:
I figured that made the line a bit long (91 chars), but PEP 257 doesn't say anything about line lengths. 🤷‍♂️